modules/sdk-coin-bsc/test/unit/utils.ts

```diff
@@ -0,0 +1,29 @@
+import assert from 'assert';
+import should from 'should';
+import { NetworkType } from '@bitgo/statics';
+import { getCommon } from '../../src/lib/utils';
+
+describe('Network Common Configuration', () => {
+  it('getCommon for mainnet', () => {
+    const common = getCommon(NetworkType.MAINNET);
+    should.equal(common.chainName(), 'mainnet');
+    should.equal(common.hardfork(), 'petersburg');
```

Comment:
Not too sue about the hardfork, but used `petersburg` since most other eth-likes used that.